packages/models/src/targets.ts

```diff
@@ -28,7 +27,11 @@ export class Targets extends Signed {
     this.delegations = options.delegations;
   }
 
-  public equals(other: Targets): boolean {
+  public addTarget(target: TargetFile): void {
+    this.targets[target.path] = target;
+  }
+
```

Comment:
New function to support the addition of targets to the targets-metadata. Needed by the new `repo-mock` package.